plugins/modules/cloud/misc/terraform.py

```diff
@@ -181,20 +181,26 @@
 
 from ansible.module_utils.basic import AnsibleModule
 
-DESTROY_ARGS = ('destroy', '-no-color', '-force')
-APPLY_ARGS = ('apply', '-no-color', '-input=false', '-auto-approve=true')
 module = None
 
 
-def preflight_validation(bin_path, project_path, variables_args=None, plan_file=None):
+def get_version(bin_path):
+    extract_version = module.run_command([bin_path, 'version', '-json'], use_unsafe_shell=True)
+    transform_version = (json.loads(extract_version[1]))['terraform_version']
```

Comment:
Accept my apologies, I don't user terraform myself, is `transform` something related to that? Or should this be `terraform_version`?

Reply:
this is the returned key from terraform

Reply:
My question is about `transform_version`, why is that named `transform_version` instead of `terraform_version`?
```
$ terraform version -json
{
  "terraform_version": "0.15.0",
  "platform": "linux_amd64",
  "provider_selections": {},
  "terraform_outdated": false
}
```

Reply:
ah yes thank you for your improvements :-)